src/database/schema/ChatSchema.ts

```diff
@@ -312,7 +324,7 @@ class ChatSchema implements BaseDbSchema {
       rawJson,
       status,
       isContinuous: false,
-      type: 'regular',
+      type: data?.message?.message_type ?? data?.message?.type,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is to handle the case where `message_type` might not be present in `data.message`. It's important to ensure that the fallback value (`data?.message?.type`) is always available or else it could lead to unexpected behavior.
```diff
-      type: 'regular',
+      type: data?.message?.message_type ?? data?.message?.type,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->